### PR TITLE
Feature/tabs in modals

### PR DESCRIPTION
**Pull request by Bluesmile82**
Modals could show only one metadata source at a time. There were some cases (i.e country GHG emissions) where we needed to display two different sources in tabs with a custom title.

- [x] Add tabs to the modal header
- [x] Add custom title to tab provided as a param to setModalMetadataParams

image

---
Some working info modals: country GHG emissions, GHG emissions page

### CODE REVIEW

**Comment by edbrett** [comment]
nov-15-2017 16-40-06

**Comment by Bluesmile82** [comment]
Good catch! Fixed

File: app/javascript/app/components/modal/modal-header-component.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import PropTypes from 'prop-types';
+import Tab from 'components/tab';
+
+import styles from './modal-styles.scss';
+
+class ModalHeader extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      selectedIndex: 0
```
Comment (j8seangel) [inline app/javascript/app/components/modal/modal-header-component.jsx]: We are not using this, aren't we?

File: app/javascript/app/components/ghg-emissions/ghg-emissions.js [hunk 2]
```diff
@@ -183,7 +183,7 @@ class GhgEmissionsContainer extends PureComponent {
     const { source } = this.props.sourceSelected;
     if (source) {
       this.props.setModalMetadata({
-        slug: source,
+        slugs: [source],
```
Comment (j8seangel) [inline app/javascript/app/components/ghg-emissions/ghg-emissions.js]: I'd rather prefer to check the method if is a string or an array than set an array as mandatory param.

**Comment by Bluesmile82** [comment]
Ready! Thanks

**Comment by edbrett** [comment]
Looks good to me. @j8seangel can you confirm the last changes?

**Comment by edbrett** [comment]
Great stuff.

**Pull request merged**